sdk/src/main/java/com/tmsdurham/actions/ResponseBuilder.kt

```diff
@@ -240,6 +240,26 @@ data class RichResponse(
 }
 
 
+enum class ImageDisplayOption {
```

Comment:
The official SDK uses the name `ImageDisplays`.  For maintainability and readability reasons, we want to keep naming identical with the official SDK, even though it might be a bit against naming conventions for Java enums (plural naming is not the convention).  Also, please match the relative order of the official SDK, which places this above SimpleResponse.